README.md

```diff
@@ -7,6 +7,25 @@ The goal of this project is to
 1. train a model to differentiate between the gene expression of healthy individuals and individuals with a disease
 2. apply the model to rare diseases and find how gene expression differs in the disease state
 
+## Method Overview
+The current plan is to use [PLIER](https://github.com/wgmao/PLIER) as a dimensionality reduction method to help the training of a model that
+predicts whether gene expression is healthy or unhealthy. 
+PLIER is well suited for this problem, because it uses prior biological knowlege to constrain the embedding.
+This classifier will then be used to make predictions about diseases, hopefully classifying their gene expression as unhealthy.
+Finally, some form of model interpretation will determine which of the biological networks used by PLIER most strongly indicate to the classifier that something is wrong.
+
+Some patients don't ever receive a diagnosis explaining what their rare disease is, and a method like brd-net would help guide doctors and researchers to find the right diagnosis.
+
+## Data Generation
+As there is not a database (that we know of) that contains labels denoting whether samples in the [Sequence Read Archive](https://www.ncbi.nlm.nih.gov/sra)
```

Comment:
You could explain what type of data is in SRA - human? gene expression? healthy or disease?